friends/src/components/NavBar.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import { NavLink } from 'react-router-dom'
+import './NavBar.css';
+
+function NavBar(props) {
+    return(
+        <div>
+            <NavLink to ="/"><h1>Friends</h1></NavLink>
+            <div  className="home">
+                <NavLink to="/">
+                    <div>Home</div>
+                </NavLink>
+                <NavLink to="/friend-form">
+                    <div className="add">{props.isUpdating ? "Update Friend" : "Add Friend"}</div>
```

Comment:
Consider deleting the logic on line 14 or making it so that isUpdating goes false if a person clicks 'Home' before finishing the update.